package.json

```diff
@@ -5,9 +5,9 @@
   "main": "index.js",
   "scripts": {
     "dev": "better-npm-run dev",
-    "migrate": "sequelize db:migrate",
-    "migrate:reset": "sequelize db:migrate:undo:all && npm run migrate",
-    "seed": "sequelize db:seed:all",
+    "migrate": "sequelize db:migrate --name 20230206125837-create-user.js",
+    "migrate:reset": "sequelize db:migrate:undo --name 20230206125837-create-user.js && npm run migrate",
+    "seed": "sequelize db:seed --seed 20230206145743-User.js",
```

Comment:
Don't do this. What happens if we have multiple migrations or seedings? Let's use the previous scripts.

Reply:
For this, circle ci is trying to access other migrations named 'create-jwt-tokens' which are from a different task. Which is causing it to fail!